lib/ASN1/Universal/Integer.php

```diff
@@ -42,15 +42,20 @@ public function getContent()
         return $this->value;
     }
 
+    public function rightShift($number, $positions)
```

Comment:
I there a reason why this method is public? Seems to me like this should be private.